model/rpc/registry.yaml

```diff
@@ -40,28 +40,48 @@ groups:
           For example, a property `my-custom-key` with value `["attribute_value"]` SHOULD be recorded as
           the `rpc.response.metadata.my-custom-key` attribute with value `["attribute_value"]`
         examples: [["attribute_value"]]
-      - id: rpc.method
+      - id: rpc.method.name
```

Comment:
alternative (closer to HTTP): `rpc.request.method` (and `rpc.request.method_original`)